datamade/committee-oversight
# Comparison pages

## Overview

Closes #114. This PR creates committee comparison pages to match the following 2 mockups:

1. https://app.moqups.com/NUARyAts8L/edit/page/a30a4e8a9
2. https://app.moqups.com/NUARyAts8L/edit/page/a30932342

## Testing Instructions

* Set up the site as directed in the README (the cms import includes a few new pages)
* Log in as an admin and check the following urls to make sure they match up with the mockups above and look good at various sizes:
	- http://localhost:8000/compare-current-committees/
	- http://localhost:8000/compare-committees-over-congresses/
* Check that the new links on the landing page work
